scripts/update_lints.py

```diff
@@ -70,8 +92,37 @@ def get_clippy_lints():
     return merged_lints
 
 
-if __name__ == "__main__":
-    output = [{"name": l[0], "group": l[1], "rustc": True} for l in get_rustc_lints()] + \
-             [{"name": l[0], "group": l[1], "rustc": False} for l in get_clippy_lints()]
+def generate_text(lints_list: list, hardcoded_part: str):
+    lints_list.sort(key=lambda item: (not item[1], item[0]))
+    beautiful_list = []
+    for i in lints_list:
+        beautiful_list.append("    Lint(\"{}\", {})".format(i[0], str(i[1]).lower()))
+    text = hardcoded_part + ",\n".join(beautiful_list) + "\n)\n"
+    return text
+
+
+class LintsUpdater(UpdaterBase):
+
+    def _update_locally(self):
+        text_rustc = generate_text(get_rustc_lints(), HARDCODED_PART_RUSTC)
+        with open(RUSTC_LINTS_PATH, "w") as f:
+            f.write(text_rustc)
```

Comment:
Why do not put this code into `generate_text` as well? Of course, with renaming `generate_text` to something more suitable?

Reply:
Now I don't want to put everything in one function, because then the function returns only text, it is much easier to check it during making changes (like adding new lines/spaces/etc in the resulting text). 

Creating a new function like `write_text(text, file)`  will not greatly affect readability and will not reduce the number of lines.

Reply:
I'd still move writing to file into this function, but it's up to you